support_bundle/get_support_bundle.sh

```diff
@@ -202,6 +202,47 @@ main() {
         get_agent_version_metric_limits
     fi
 
+    # If Secure API key is supplied, collect settings
+    if [[ ! -z ${SECURE_API_KEY} ]]; then
+        VERSION_CHECK=$(kubectl ${KUBE_OPTS} get cm | grep -c 'sysdigcloud-api-config') || true
```

Comment:
can you use the BACKEND_VERSION from your other PR here instead of doing this check again?